src/authorizeStep.js

```diff
@@ -14,6 +14,7 @@ export class AuthorizeStep {
     if (routingContext.getAllInstructions().some(i => i.config.auth)) {
       if (!isLoggedIn) {
         console.log("login route : " + loginRoute);
```

Comment:
I know it's not yours, but this is the first time I notice it. It's a console.log and doesn't belong in the code, so I just asked if you'd be willing to take it out for us :)